_sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md

```diff
@@ -0,0 +1,20 @@
+---
+type: watm-event
+title: connection-practice-meeting
+name: Connection Practice Meeting
+facilitator: Michelle Kelley
+day: 1
+date: '2020-06-12'
+start-time: '17:00'
+end-time: '18:00'
+link: TBA
+calendar-url:
```

Comment:
`calendar-link` right?

Reply:
Yes, thank you. I'll make these consistent -- not sure how they ended up not.